### ISSUE TITLE
Pads not listed

### ISSUE THREAD

**Issue by binarydad** [rhansen/ep_adminpads2#14]
I've installed this version from the plugins page and the pads I have created are not displaying. I'm using 1.8.4~ynh1 on Yunohost. 

![image](https://user-images.githubusercontent.com/1760607/93921525-a89ca980-fcde-11ea-9e0b-19b027f7cce6.png)
![image](https://user-images.githubusercontent.com/1760607/93921549-b2bea800-fcde-11ea-9843-b165528fd00d.png)

**Comment by rhansen** [comment]
Hmm... Not sure what could be causing that. Questions:
  * Has this plugin ever worked for you? If so, please try downgrading to the version of ep_adminpads2 that worked.
  * Does it work if you restart your Etherpad server?
  * Does it work if you upgrade Etherpad to the latest commit on the develop branch?
  * Do any error messages show up in the JavaScript console in your browser?
  * Can you post a server log with debug info?

**Comment by Sh0uld** [comment]
Not sure, if it's the same issue, but on my Etherpad server the version 2.1.0 isn't listing the pads, too. 2.0.0

**Comment by rhansen** [comment]
Thank you for providing more information @Sh0uld. That TypeError is weird—I don't understand why it's happening. I tweaked the code a bit to hopefully avoid that error and published v2.1.1. Please try it out and let me know how it works.

**Comment by Sh0uld** [comment]
Hi @rhansen,

thank you for your support! Unfortunately it does not solve this issue.

I've looked a bit in the code myself, but I'm new to nodejs. The variable query itself is null and this is causing the error. I've tried a few things, which doesn't help:

- add checks for `query != null`. There I ended in line 45, there the variable query is used without an if statement.
- check for `query != null` and initialise query at the beginning with `query = {pattern: '', offset: 0, limit: queryLimit};`. Then the error disappeared on server side, but on client side I get this error (in firefox): `Uncaught TypeError: data.query is null`

But version 2.0.0 is still working fine. Maybe review the changes in 2.1.0?

**Comment by rhansen** [comment]
I think I figured out what's going wrong, but I can't reproduce this bug so I'm not sure.

Would you please try #15? You can install it by running:

```shell
cd /path/to/etherpad-lite
npm i "rhansen/ep_adminpads2#client-load"
```

**Comment by Sh0uld** [comment]
On my side works everything fine now - thank you!

**Comment by rhansen** [comment]
I published v2.1.2. Thanks for testing @Sh0uld!

@binarydad: If v2.1.2 doesn't work for you please let me know and I'll do some more troubleshooting.
